volttron/platform/vip/agent/subsystems/ping.py

```diff
@@ -82,6 +82,8 @@ def ping(self, peer, *args):
         result = next(self._results)
         args = list(args)
         args.insert(0, b'ping')
+        if peer == b'pubsub':
+            peer = b''
```

Comment:
I think moving publish to self.vip.publish is not a good idea. It breaks backward comparability.